frontend-public/src/components/landingPage/LandingPage.js

```diff
@@ -1,53 +1,40 @@
 import React from "react";
-import { Divider, Row, Col, Button } from "antd";
+import { Divider, Button } from "antd";
 
-import QuestionSidebar from "./common/QuestionsSidebar";
+import QuestionSidebar from "../common/QuestionsSidebar";
```

Comment:
I'm a bit leery of the relative path here since we have the `@` alias and use absolute paths pretty much everywhere else

Reply:
fixed.